tests/integration/test_stripe.py

```diff
@@ -17,63 +16,59 @@ class DescribeStripeAssembler:
 
     def it_provides_values_for_univariate_cat(self):
```

Comment:
I guess this test should be moved in test_cubepart, or maybe this is not needed anymore cause in the test_cubepart (integration) we do have something similar (to be checked)